Source/WebKit/GPUProcess/graphics/RemoteImageBuffer.h

```diff
@@ -42,7 +42,7 @@ class RemoteImageBuffer : public WebCore::ImageBuffer {
     template<typename BackendType>
     static RefPtr<RemoteImageBuffer> create(const WebCore::FloatSize& size, float resolutionScale, const WebCore::DestinationColorSpace& colorSpace, WebCore::PixelFormat pixelFormat, WebCore::RenderingPurpose purpose, RemoteRenderingBackend& remoteRenderingBackend, QualifiedRenderingResourceIdentifier renderingResourceIdentifier)
     {
-        auto context = ImageBuffer::CreationContext { nullptr
```

Comment:
Hmm why was the `WebCore::` prefix not needed before? A stray `using namespace WebCore` in a header somewhere?